board/recalbox/fsoverlay/etc/init.d/S02splash

```diff
@@ -3,8 +3,6 @@
 case "$1" in
   start)
       fbv -f -i /recalbox/system/resources/splash/logo-loading.png &
-      sleep 1
```

Comment:
No, it doesn't.
It's not documented, but i can read the code for a confirmation.


2016-01-04 10:50 GMT+01:00 digitalLumberjack <notifications@github.com>:

> In board/recalbox/fsoverlay/etc/init.d/S02splash
> <https://github.com/recalbox/recalbox-buildroot/pull/107#discussion_r48715964>
> :
>
> > @@ -3,8 +3,6 @@
> >  case "$1" in
> >    start)
> >        fbv -f -i /recalbox/system/resources/splash/logo-loading.png &
> > -      sleep 1
>
> fbv doesn't stay alive if we do not killall ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/recalbox/recalbox-buildroot/pull/107/files#r48715964>.
>